### ISSUE TITLE
Cells not rendered clearly by moxunit_util_elem2str on MATLAB

### ISSUE THREAD

**Issue by scottclowe** [MOxUnit/MOxUnit#15]
When using `moxunit_util_elem2str` on MATLAB, cells are rendered pretty much indistinguishably from their contents.

Example:
```
>> moxunit_util_elem2str('bar')
ans =
'bar'
>> moxunit_util_elem2str({'bar'})
ans =
    'bar'
```

An empty cell renders nothing at all (returned as an empty string).

**Comment by nno** [comment]
How would you propose to improve this? How about adding datatype and size as here:

```
>> moxunit_util_elem2str('bar')
ans =
'bar'
>> moxunit_util_elem2str({'bar'})
ans =
       'bar' (1x1 cell)
```

The datatype and size could be added only for singleton, non-numeric elements, so that it is not shown for larger inputs, or for scalars.

**Comment by scottclowe** [comment]
Maybe it would be best to always show the type and size (along with the content for small elements as is currently implemented)?

It seems to me it would have a low cost to do this, since it is always a small number of characters, it would make the output more consistent since this is shown for larger outputs, and the size and class information may prove useful in circumstances we have not foreseen, such someone comparing a 'bar' string in the regular character class with another 'bar' whose type is some user-created subclass of the built-in character class.

**Comment by nno** [comment]
@scottclowe as you suggested, the type and size are now (as of merged #32) always shown. I'm closing the issue; feel free to re-open if you think functionality is missing.
